gridded_etl_tools/utils/attributes.py

```diff
@@ -219,6 +219,18 @@ def missing_value_indicator(cls) -> str:
     Tags for dataset.
     """
 
+    dataset_category: str = "observation"
```

Comment:
You should use `typing.Literal` instead of `str` for the type, since we want to clue the user into the fact that we want them to use a limited number of strings when overwriting